kernel/Kconfig.power_mgmt

```diff
@@ -49,7 +49,7 @@ config SYS_POWER_LOW_POWER_STATE
 	  saving most power.
 
 config SYS_POWER_DEEP_SLEEP
```

Comment:
Since we are reworking names I think we should use the same convention for both low power and deep sleep. I.e. we should choose one of the pairs:
- `SYS_POWER_LOW_POWER`, `SYS_POWER_DEEP_SLEEP`
- `SYS_POWER_LOW_POWER_STATES`, `SYS_POWER_DEEP_SLEEP_STATES`